### DISCUSSION TITLE
Autocompletion of files' paths doesn't work

### DISCUSSION THREAD

**Discussion by RegreTTO** [microsoft/pylance-release#3699] [Enhancement]
## Environment data

-   Language Server version: Pylance language server 2022.11.41 (pyright 621d886b)
-   OS and version: Arch Linux, Kernel: 6.0.10-zen2-1-zen
-   Python version (& distribution if applicable, e.g. Anaconda): 3.9.13 Anaconda

## Code Snippet

![изображение](https://user-images.githubusercontent.com/39791198/205096922-9817a9a5-8e4f-4f1b-9766-a1304f16b933.png)

## Repro Steps

1. Create some data files
2. Type something like ``` open("<path>")```

## Expected behavior 

I get path suggestion
(I switched to Jedi server to show you the expected behavior)

![изображение](https://user-images.githubusercontent.com/39791198/205097670-3cbba9d3-ceb3-4e66-b781-cf1862de8647.png)

## Actual behavior

Nothing shows

## Logs

https://gist.github.com/RegreTTO/51d402c077f46173f51a88d3e18a2978
(Github doesn't allow such big amount of text, as I get in logs)

**Comment by diego351** [comment]
Can confirm
